src/utils/formatting-doc-to-txt.ts

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+
+const MAX_ANSWERS = 8;
+
+const inputFilename = 'input.txt';
+const outputFilename = 'output.csv';
+
+// Read the input file and split it into lines
+const lines = fs.readFileSync(inputFilename, 'utf8').split('\n');
+
+// Create an array to hold the data for the output CSV file
+var outputData = [];
+
+// Initialize the line index
+var i = 0;
+
+while (i < lines.length) {
+    // Check if this is a question line
+    if (lines[i].includes('.')) {
+        var question = lines[i].trim().replace(',', '');
```

Comment:
var is deprecated, should use const or let, read more [here ](https://plainenglish.io/blog/4-reasons-why-var-is-considered-obsolete-in-modern-javascript)

For this code, everywhere var should be replaced with const except in the case of `var i = 0` that should be let `i = 0`